extension/war/src/main/resources/locale/portal/HamburgerMenu_en.properties

```diff
@@ -22,4 +22,6 @@ menu.spaces.exploreSpaces=Explore Spaces
 menu.spaces.noSpacesFound=No spaces found
 menu.productName.seeProduct=See product updates
 menu.settings=Access User Settings
-menu.logout=Log out
\ No newline at end of file
+menu.logout=Log out
+menu.settings.label=Settings
+menu.logout.label=Logout
```

Comment:
Why adding a new label as menu.logout is existing and the same one ?

Reply:
Used the same labels for settings and logout